interfaces/builtin/libvirt.go

```diff
@@ -23,6 +23,7 @@ import "github.com/snapcore/snapd/interfaces"
 
 const libvirtConnectedPlugAppArmor = `
 /run/libvirt/libvirt-sock rw,
+/etc/libvirt/* r,
 `
```

Comment:
FYI, the /etc/libvirt/* rule is fine since the libvirt interface is an implicit classic slot.